requirements.txt

```diff
@@ -3,3 +3,4 @@ jupyterlab>=3.5.1
 numpy>=1.23.5
 pandas>=1.5.2
 plotly>=5.11.0
+pydot>=1.3.0
```

Comment:
Is this needed? where is it being used?

Reply:
Let me check if this is required or not, it is being imported in execution trace file so I added it

Reply:
Is it being added to execution_trace.py or are you referring to some other file?

Reply:
execution_trace.py uses param -> which uses pydot import. I tried and we cannot run the notebook without it